content/outreach/struc2drug.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Struc2drug seminar series
+date: 2020-03-06
+publishdate: 2020-03-06
+menu:
+    main:
+        parent: Education
+        name: Struc2drug
+weight: 100
+nav: false
+summary: |
+    Struc2drug is a bimonthly seminar series promoting the exchange between researchers in the interdisciplinary field
+    of structural biology and drug development in Berlin.
+---
+
+Drug design is a complex and lengthly process that can only yield successful results using a plethora of different
+experimental and computational methods. Many researchers work in Berlin at universities, research facilities and
+industry in the field of drug design or in neighboring disciplines.
+
+*Struc2drug* is a bimonthly seminar series promoting the exchange between researchers in the interdisciplinary field
+of structural biology and drug development in Berlin.
+
```

Comment:
Add here the struc2drug logo?

`{{< xfigure src="/images/research/struc2drug.png" imageclass="fit" >}}`